kplutus-pyk/src/kplutus_pyk/__main__.py

```diff
@@ -1,36 +1,67 @@
-from argparse import ArgumentParser
+import sys
+from argparse import ArgumentParser, ArgumentTypeError
+from pathlib import Path
 
 from pyk.cli_utils import dir_path, file_path
 
 from .kplutus import KPlutus
 
 
 def main() -> None:
+    sys.setrecursionlimit(15000000)
+
     parser = create_parser()
     args, remainder = parser.parse_known_args()
 
     if args.command == 'kompile':
         KPlutus.kompile(args.main_file, remainder)
     if args.command == 'uplc-to-k':
-        KPlutus.uplc_to_k(args.main_file, args.definition, remainder)
+        kplutus = KPlutus(args.kplc_lib)
+        if not args.definition:
+            args.definition = kompiled_dir(".")
```

Comment:
```suggestion
            args.definition = kompiled_dir('.')
```

There's a mixed use of single and double quotes in the code. I prepared a PR (https://github.com/runtimeverification/plutus-core-semantics/pull/352) that adds a dev dependency `flake8-quotes` that will flag these inconsistencies. Consider merging that PR before this one.